install/kubernetes/helm/istio/test-values/values-e2e.yaml

```diff
@@ -15,6 +15,10 @@ global:
   outboundTrafficPolicy:
     mode: REGISTRY_ONLY
 
+pilot:
+  env:
+    PILOT_ENABLE_PROTOCOL_SNIFFING: true
```

Comment:
@nmittler FYI

Reply:
OK great .... so we don't need to update our yaml transformations.  Good to know.

The same thing will be possible with the new integration test framework, which allows you to manually provide values overrides.

Reply:
My concern is that it's very hard to cover all the cases which already exists in pilot test.